recipes/protoc/3.9.x/conanfile.py

```diff
@@ -26,6 +28,10 @@ def _build_subfolder(self):
     def _cmake_base_path(self):
         return os.path.join("lib", "cmake", "protoc")
 
+    def configure(self):
+        if self.settings.os_build != self.settings.os and self.settings.arch_build != self.settings.arch:
```

Comment:
Nope, we need to support it.

Reply:
The new crossbuilding pr by jsgogo will fix this.

Reply:
Yes! :clap:

Reply:
So let's wait with this pr until that pr lands?
Because that might take some time.

Reply:
well, we can push it forward, and update once that feature is available.

Reply:
Ok. Can you review and merge it?
(and maybe rebase on top of CCI's master since this branch is quite old)